src/pages/error/EntityNotFound.tsx

```diff
@@ -11,6 +14,12 @@ const EntityNotFound = ({ catalogName }: Props) => {
         header: 'routeTitle.error.entityNotFound',
     });
 
+    useMount(() => {
+        logRocketEvent(CustomEvents.ENTITY_NOT_FOUND, {
+            catalogName,
+        });
+    });
```

Comment:
Added the event as we are now validating catalog names on details and want to be safe... but also makes sense to track if this component is shown.